org.openomf.OpenOMF.json

```diff
@@ -0,0 +1,191 @@
+{
+  "app-id": "org.openomf.OpenOMF",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openomf",
+  "permissions": [
+      "xdg-config",
+      "xdg-state"
+  ],
+  "finish-args": [
+    "--share=network",
```

Comment:
why does this need network access?

Reply:
Network play? I can remove it for now as this version has it disabled, but the next major release will have network play enabled.

Reply:
if it doesn't have then it doesn't need.

Reply:
Removed.